ecole-doctorale-de-sciences-po-author-date-french.csl

```diff
@@ -0,0 +1,440 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="fr-FR" version="1.0" page-range-format="expanded" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Ecole doctorale de Sciences Po (author-date, French)</title>
+    <id>http://www.zotero.org/styles/ecole-doctorale-de-sciences-po-author-date-french</id>
+    <link href="http://www.zotero.org/styles/ecole-doctorale-de-sciences-po-author-date-french" rel="self"/>
+    <author>
+      <name>Anna Couthures-Idrizi</name>
+      <email>anna.couthuresidrizi@sciencespo.fr</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <category field="generic-base"/>
+    <updated>2019-01-10T00:59:23+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="ordinal-01">ère</term>
+      <term name="ordinal-02">e</term>
+      <term name="ordinal-03">e</term>
+      <term name="ordinal-04">e</term>
+      <term name="cited">Op.&#160;cit.</term>
+      <term name="page" form="short">p.</term>
+	<term name="editor" form="short">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+<macro name="contributors-note">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name form="short" and="text" delimiter-precedes-last="never" sort-separator=" " font-style="normal">
+          </name>
+        </names>
```

Comment:
let's handle this with `substitute`, please

Reply:
i don't understand how to do that

Reply:
could you help me, please ?